README.md

```diff
@@ -15,6 +15,12 @@ feed](https://azure.microsoft.com/updates/).
 Users may prefer to jump right in to our samples repo at
 [github.com/Azure-Samples/azure-sdk-for-go-samples][samples_repo].
 
+Questions and feedback? Chat with us in the **[#Azure SDK
+channel](https://gophers.slack.com/messages/CA7HK8EEP)** on the [Gophers
+Slack](https://gophers.slack.com/). Sign up
+[here](https://invite.slack.golangbridge.org) first if necessary.
```

Comment:
only thing is i would have added some more info in the readme about using the interfaces for mocking, but i guess its too late ?

Reply:
I will add a sample for this.